### PR TITLE
Added new names

### PR DESCRIPTION
**Pull request by STeeL835**
## Overview

Added Hideo Kojima and William Dafoe to the list

## Before merge

Check that
- [x] Tests are green
- [x] Code review is performed
- [x] All comments are resolved

### CODE REVIEW

File: names.txt [hunk 1]
```diff
@@ -3,4 +3,6 @@ A2 from NieR: Automata
 Detective Cole Phelps from L.A. Noire
 Mustache Girl from A Hat in Time
 Senua from Hellblade: Senua's Sacrifice
-Niko from OneShot
\ No newline at end of file
+Niko from OneShot
+William Dafoe
+Hideo Kodzima
```
Comment (STeeL835) [inline names.txt]: I think Kojima is spelled wrong

File: names.txt [hunk 2]
```diff
@@ -3,4 +3,6 @@ A2 from NieR: Automata
 Detective Cole Phelps from L.A. Noire
 Mustache Girl from A Hat in Time
 Senua from Hellblade: Senua's Sacrifice
-Niko from OneShot
\ No newline at end of file
+Niko from OneShot
+William Dafoe
```
Comment (STeeL835) [inline names.txt]: It's Willem Dafoe

**Comment by STeeL835** [comment]
How about adding Keanu Reeves, everyone loves him

File: names.txt (hunk 1 above)
Comment (STeeL835) [reply names.txt]: Ok, I'll change it

**Review by NulleanAndRain — APPROVED** [review]
Nice

**Pull request merged**